### PR TITLE
Test notebook in CI

### PR DESCRIPTION
**Pull request by AntoinePrv**
- Run notebook without error
- Simplify GHA workflow
- Run notebook in CI

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -24,57 +25,46 @@ jobs:
...
 
-    - name: install xwidgets
-      shell: bash -l {0}
-      run: |
-        cd build
-        make install
+    - name: Test notebook
+      if: ${{ startsWith(matrix.os, 'ubuntu') }}
+      shell: bash -l -eo pipefail {0}
+      run: papermill notebooks/xwidgets.ipynb "$(mktemp -d)/out.ipynb"
```
Comment (AntoinePrv) [inline .github/workflows/main.yml:65]: Addition where we test the example notebook.

**Comment by SylvainCorlay** [comment]
This appears to conflict with the main branch.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -99,34 +89,30 @@ jobs:
...
-         -DCMAKE_BUILD_TYPE=Release ^
-         -DXWDIGETS_BUILD_TESTS=ON ^
-         -DCMAKE_PREFIX_PATH="%CONDA_PREFIX%\Library" ^
-         -DCMAKE_INSTALL_PREFIX="%CONDA_PREFIX%"
-      working-directory: build
-
-    - name: build
-      shell: cmd
+         -D CMAKE_BUILD_TYPE=Release ^
+         -D XWIDGETS_BUILD_TESTS=ON ^
```
[the remark this answers is not in the document]
Comment (SylvainCorlay) [reply .github/workflows/main.yml:100]: Wow good catch.

File: .github/workflows/main.yml (hunk 1 above)
Comment (AntoinePrv) [reply .github/workflows/main.yml:65]: Surely, I did not know of that possibility.

**Pull request merged**